plugins/module_utils/redfish_utils.py

```diff
@@ -2625,14 +2625,14 @@ def set_manager_nic(self, nic_addr, nic_config):
         target_ethernet_current_setting = None
         if nic_addr == 'null':
             # Find root_uri matched EthernetInterface when nic_addr is not specified
-            nic_addr = (self.root_uri).split('/')[-1]
+            nic_addr = (self.root_uri).lower().split('/')[-1]
```

Comment:
This lowercases the nic_addr that is derived from the root_uri if an explicit nic_addr is not provided.
Suggest also making the explicitly passed nic_addr lowercase.

Reply:
Good point.

To be honest I'm not that familiar with the code base and was kinda hoping for someone more experienced to hop in. If there's no better place than I'm glad to help:

https://github.com/ansible-collections/community.general/pull/1744/commits/0927df2a29da6bbf3d1723663d771b35e3f38e4e#diff-d95ef80c503a120e81b8838cf7cf97971efcd310ee5c97b301b60bbf3ad07b8bR2635

Reply:
@billdodd Do you think my approach is acceptable and we can resolve this discussion?

Reply:
Yes, the change LGTM. Thanks again for your help.